docs/UML diagrams/AddIngredientSequence.puml

```diff
@@ -0,0 +1,38 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+[-> ":AddCommand": new AddCommand("egg", 5)
+
+activate ":AddCommand"
+
+":AddCommand" -> ":IngredientList": getIngredients()
+activate ":IngredientList"
+":AddCommand" <- ":IngredientList": ArrayList<Ingredient>: ingredients
+deactivate ":IngredientList"
+
+":AddCommand" -> ":AddCommand": findIndex("egg")
+activate ":AddCommand"
+
+":AddCommand" -> ":AddCommand": return -1
+
+deactivate ":AddCommand"
+
```

Comment:
Can use an X at the end of the lifeline of an object to show its deletion for AddCommand.